satviz-serial/src/test/java/edu/kit/satviz/serial/ByteArrayInputStream.java

```diff
@@ -0,0 +1,24 @@
+package edu.kit.satviz.serial;
+
+import java.io.InputStream;
+
+public class ByteArrayInputStream extends InputStream {
```

Comment:
Mit `ByteArrayInputStream` und `ByteArrayOutputStream` scheinen zwei Klassen implementiert worden zu sein, die es gleichnamig bereits in der Standardbibliothek gibt?